pgjdbc/src/test/java/org/postgresql/test/jdbc3/GeneratedKeysTest.java

```diff
@@ -198,7 +200,7 @@ public void testMultipleRows() throws SQLException {
   public void testSerialWorks() throws SQLException {
     Statement stmt = con.createStatement();
     int count = stmt.executeUpdate(
-        "INSERT INTO genkeys (b,c) VALUES ('a', 2), ('b', 4)" + returningClause + "; ",
+        "INSERT/*fool parser*/ INTO genkeys (b,c) VALUES ('a', 2), ('b', 4)" + returningClause + "; ",
```

Comment:
can we handle a nested comment ?

Reply:
`parseBlockComment` does handle that case as well. I've added a test case just in case

Reply:
nice!